process/physics.py

```diff
@@ -5752,6 +5933,276 @@ def bootstrap_fraction_sakai(
+            - A = 1.5, R_0 = 0.3m, I_p = 200kA, B_0=0.4T, beta = 4-10%. Profiles taken as Gaussian shaped functions.
+            - Errors mostly up to the order of 10% are obtained when both expressions are compared with the equilibrium estimates for the
+              bootstrap current in ETE
+
+        References:
+            - M. C. R. Andrade and G. O. Ludwig, “Scaling of bootstrap current on equilibrium and plasma profile parameters in tokamak plasmas,”
+              Plasma Physics and Controlled Fusion, vol. 50, no. 6, pp. 065001–065001, Apr. 2008,
+              doi: https://doi.org/10.1088/0741-3335/50/6/065001.
+
+        """
+
+        # Using the standard variable naming from the Andrade et.al. paper
+        c_p = core_pressure / average_pressure
+
+        # Error +- 0.0007
+        c_bs = 0.2340
+
+        return c_bs * np.sqrt(inverse_aspect) * betap * c_p**0.8
+
+    @staticmethod
+    def bootstrap_fraction_hoang(
+        betap: float,
+        pressure_index: float,
+        current_index: float,
+        inverse_aspect: float,
+    ) -> float:
+        """
+        Calculate the bootstrap fraction using the Hoang et al formula.
+
+        Parameters:
+        betap (float): Plasma poloidal beta.
+        pressure_index (float): Pressure profile index.
+        current_index (float): Current profile index.
+        inverse_aspect (float): Inverse aspect ratio.
+
+        Returns:
+        float: The calculated bootstrap fraction.
+
+        Notes:
+            - Based off of TFTR data calculated using the TRANSP plasma analysis code
+            - 170 discharges which was assembled to  study the tritium influx and transport in discharges with D-only neutral beam
+              injection (NBI)
+            - Contains L-mode, supershots, reversed shear, enhanced reversed shear and increased li discharges
+            - Discharges with monotonic flux profiles with reversed shear are also included
+            - Is applied to circular cross-section plasmas
+
+        References:
+            - G. T. Hoang and R. V. Budny, “The bootstrap fraction in TFTR,” AIP conference proceedings,
+              Jan. 1997, doi: https://doi.org/10.1063/1.53414.
+        """
+
+        # Using the standard variable naming from the Hoang et.al. paper
+        # Hoang et.al uses a different definition for the profile indexes such that
+        # alpha_p is defined as the ratio of the central and the volume-averaged values, and the peakedness of the density of the total plasma current
+        # (defined as ratio of the central value and I_p), alpha_j$
+
+        # We assume the pressure and current profile is parabolic and use the (profile_index +1) term in lieu
+        # The term represent the ratio of the the core to volume averaged value
+
+        # This could lead to large changes in the value depending on interpretation of the profile index
+
+        c_bs = np.sqrt(pressure_index + 1 / current_index + 1)
```

Comment:
Should it not be this?

```suggestion
        c_bs = np.sqrt((pressure_index + 1) / (current_index + 1))
```